player/command.c

```diff
@@ -5321,6 +5321,45 @@ static void cmd_playlist_next_prev(void *p)
         mpctx->add_osd_seek_info |= OSD_SEEK_INFO_CURRENT_FILE;
 }
 
+static void cmd_playlist_next_prev_playlist(void *p)
+{
+    struct mp_cmd_ctx *cmd = p;
+    struct MPContext *mpctx = cmd->mpctx;
+    int direction = *(int *)cmd->priv;
+
+    struct playlist_entry *entry =
+        playlist_get_first_in_next_playlist(mpctx->playlist, direction);
+
+    if (!entry && mpctx->opts->loop_times != 1 && mpctx->playlist->current) {
+        entry = direction > 0 ? playlist_get_first(mpctx->playlist)
+                              : playlist_get_last(mpctx->playlist);
+
+        if (entry && entry->playlist_path &&
+            strcmp(entry->playlist_path,
+                   mpctx->playlist->current->playlist_path) == 0)
+            entry = NULL;
+
+        if (direction > 0 && entry && mpctx->opts->loop_times > 1) {
+            mpctx->opts->loop_times--;
```

Comment:
Don't really like how the option value changes behind the user's back, but I realize that we already do this for `mp_next_file`. Maybe this can be fixed later.

Reply:
I thought about this and it would also fix --loop-file=N not being kept when going to the next file, but to fix it we would have to introduce a whopping 3 properties corresponding to `loop-file`, `loop-playlist`, and `ab-loop-count` (e.g. `remaining-file-loops`, `remaining-playlist-loops`, `remaining-ab-loops`) which is pretty bloated.

Reply:
Well they are all different kinds of loops so it might be needed to have such a separation. I think wm4 did that for everything else that had weird overlaps like `window-scale` and `current-window-scale` and also used to silently change options, but we can discuss that in some other PR I suppose.